packages/core-components/src/layout/Sidebar/SidebarSubmenuItem.tsx

```diff
@@ -30,6 +30,8 @@ import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
 import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
 import { SidebarItemWithSubmenuContext } from './config';
 import { isLocationMatch } from './utils';
+import { Link as BackstageLink } from '../../';
```

Comment:
Avoid importing parent modules, you'll want to add `/Link` at the end of this

Reply:
:+1: